pandas/tests/series/test_reductions.py

```diff
@@ -114,3 +114,23 @@ def test_validate_stat_keepdims():
     )
     with pytest.raises(ValueError, match=msg):
         np.sum(ser, keepdims=True)
+
+
+def test_idxmax_idxmin_with_nullable_integer_dtype():
+    ser = Series([1, 2, 3], dtype="Int64")
```

Comment:
if you put these in the EA tests you can use the data_missing fixture

Reply:
Just updated the tests with parameterization.

Reply:
I think the data_missing fixture initialize an EA but not a Series, so it's not that useful here.